src/main/kotlin/racingcar/ui/ResultView.kt

```diff
@@ -1,15 +1,20 @@
 package racingcar.ui
 
-import racingcar.domain.RacingGame
-
 class ResultView {
 
-    fun printRaceProgress(racingGame: RacingGame, numberOfAttempts: Int) {
-        println("실행 결과")
-        repeat(numberOfAttempts) {
-            println("${it + 1} 회차")
-            racingGame.move()
-            racingGame.movements.forEach { println("-".repeat(it)) }
-        }
+    fun printResultTitle() {
+        println("\n실행 결과")
+    }
+
+    fun printRacingRound(round: Int) {
+        println("\n${round + 1} 회차")
+    }
+
+    fun printRacingCarMovements(names: List<String>, movements: List<Int>) {
+        names.forEachIndexed { index, name -> println("$name : ${"-".repeat(movements[index])}") }
+    }
```

Comment:
I/O는 아시다시피 굉장히 큰 비용이 드는 작업이라 적으면 적을 수록 좋은데요.
코틀린스럽게 개선한다면 아래와 같이 바꿔볼수 있으니 참고해보시면 좋을것 같습니다.

```suggestion
    fun printRacingCarMovements(names: List<String>, movements: List<Int>)= with(StringBuilder()) {
        names.forEachIndexed { index, name -> appendLine("$name : ${"-".repeat(movements[index])}") }
        print(this)
    }
```